src/main/java/leave_request/LeaveRequestRequestModel.java

```diff
@@ -0,0 +1,49 @@
+package leave_request;
+
+import entity.User;
+
+public class LeaveRequestRequestModel {
+    private User user;
+    private String message;
+    private String leaveType;  // "sick", "vacation", or "paternal"
+    private int leaveDays;
```

Comment:
Is it better to contain the startDate and endDate? Since the leaving request might processing more than 1 days.

Reply:
Or contain startDate and the leaveDays.

Reply:
You're right I'll add startDate